src/config/defaults/efi.h

```diff
@@ -37,6 +37,7 @@ FILE_LICENCE ( GPL2_OR_LATER_OR_UBDL );
 #define	SANBOOT_PROTO_IB_SRP	/* Infiniband SCSI RDMA protocol */
 #define	SANBOOT_PROTO_FCP	/* Fibre Channel protocol */
 #define	SANBOOT_PROTO_HTTP	/* HTTP SAN protocol */
+#define SANBOOT_PROTO_NBD	/* Network Block Device protocol */
```

Comment:
This should not be enabled by default for UEFI, unless there is external funding available for a Secure Boot review of the code.

Reply:
OK, fixed